src/web/blueprints/landing.py

```diff
@@ -30,8 +30,8 @@ def upload_file():
             if not file:
                 flash('No file was selected.', 'error')
                 return redirect(url_for('landing.index'))
-
             save(file, current_app.config['downloads_folder'])
+            flash('File has been successfully sent!', 'success')
```

Comment:
We can probably leave this out with the new API-based file sending, but I don't think leaving this line in will hurt?